include/oneapi/tbb/detail/_concurrent_unordered_base.h

```diff
@@ -910,6 +910,10 @@ class concurrent_unordered_base {
             // Deallocate the memory
             node_allocator_traits::deallocate(dummy_node_allocator, node, 1);
         } else {
+            // GCC 11.1 issues a warning here that incorrect destructor might be called for dummy_nodes
+            #if (__TBB_GCC_VERSION >= 110100 && __TBB_GCC_VERSION < 110200 ) && !__clang__ && !__INTEL_COMPILER
```

Comment:
Did GCC disable this option by default from 110200? Or are you going to update this file each GCC update?

Reply:
Updated to 12.0 to recheck it automatically when gcc 12.0 arrives